### ISSUE TITLE
Support Niagara Pro on Lineage / google-free devices

### ISSUE THREAD

**Issue by kr428** [8bitPit/Niagara-Issues#1482]
Having been a happy Niagara Pro user for more than a year now, I more and more consider going "google-free" on my Android device, i.e. by using a Lineage custom ROM or buying one off https://e.foundation/ . Unfortunately, so far Niagara Pro seems to be tied to Google Play store / services.
Is there a way to change that in the future? I'd really like to both keep using Niagara Pro and supporting development of the launcher, but would be great to do so on a device without Google Play services too.

**Comment by MathisSackers** [comment]
Seconding this request (I'm on CalyxOS and would love to buy the premium version)

There's two ways (I know of) that this could be achieved (I'm not an expert by any means):

1. Provide a standalone paid version of the app. This way we can buy it on the Play Store on a different phone and send the downloaded apk over. The drawback is, that it prevents automatic updates (AFAIK).
2. Provide a standalone paid app that just unlocks the premium features in the main app. Sleep As Android does this. You buy "Sleep As Android Unlock" (which is just 500kB) and the main app then checks if the Unlock is installed. This way we could buy the unlock from the Play Store, send it over to the Google-free device and still get updates for the free app via the Aurora Store (or F-Droid if it gets put on there)

**Comment by boricante** [comment]
Like you can see from the posts/likes above there is a market share that would prefer to run your launcher on custom/degoogled devices. Yes it is small but the market is there. I would implore you to consider releasing a version without Google Adsense integrated into your app even if the option is hidden on your website and/or accessible by little more complicated means. I personally wouldn't mind paying 10-20€ fee to support this product and think about all the positive publicity you would receive by making this small adjustment. Thanks for reading.

**Comment by Tifter** [comment]
I would absolutely be willing to pay a premium for this!

**Comment by 8bitPit** [comment]
Thank you all for your kind words. We're considering adding a way to unlock Niagara Pro without needing any Google Services. Still, we have to estimate the costs involved and check if that's compatible with Google Plays policies. Moreover, we have to design a user account system, search for a payment provider, and comply with tax policies worldwide. That's why this can take a while until the alternative unlock method is available.

**Comment by ozls** [comment]
Is there any news on this? Not having access to Niagara pro is really the only downside to my experience without Google. Of course an account system with a non-Google way to pay would be perfect, but in the meanwhile a separate Niagara unlocker app, like @OptionalM mentions, would be a very practical solution.

However, I would like to add another solution: is it really important to make sure people without Google Play pay for the pro feature? We're not bringing revenue at the moment anyway, and it seems unlikely anyone would go the length of removing Google from their device just to avoid paying for the feature. Unlocking Niagare Pro for devices without Google Play would be a massive marketing gesture to show users you care, to show that privacy and choice matter to you, and would presumably not take too much effort to implement.

Of course, this doesn't me I don't want to support Niagara. I have bought the Pro version before and would even be willing to donate if Niagara asked for it. Because, it is worth to say it once again, Niagara is a really great app.

**Comment by OptionalM** [comment]
This is finally solved!
[Here's the blog](https://medium.com/niagara-launcher/going-global-6bb58c2d6bb7) explaining it.
TLDR:
1. Install version 1.8.0 on your phone (Currently in Beta [here](https://help.niagaralauncher.app/article/107-beta-tester))
2. In the app click "Upgrade to Pro"
3. You'll be asked to enter an email. Your purchase will be tied to that email and you'll be able to use it on 3 devices.
4. Pay yearly or lifetime.
5. Enjoy.

If you do this before December 1st (2022) then you'll pay about $6/year or $15 for lifetime. After then it will be about double ($10/year).

@8bitPit Thank you so much! And you can close this.

**Comment by ozls** [comment]
I am absolutely delighted with this development, I am using Niagara again on my Google-free device and it feels great. Thank you so much for your attention to this issue. This issue can be closed for me as well.

**Comment by 8bitPit** [comment]
Thank you very much for your kind words @OptionalM and @ozls :D
